### DISCUSSION TITLE
Comparing two scRNA-seq data sets with each other

### DISCUSSION THREAD

**Discussion by beginner984** [satijalab/seurat#4756] [General]
I have 4 patients who are cancer free for 4 years and I have 4 controls too

I want to compare patients with controls

I used Seurat integration of my 10x PBMC data

from 6051 cells 5816 labled correctly (as per say by Seurat documentation)

<img width="454" alt="image" src="https://user-images.githubusercontent.com/43682980/125207954-8c76f280-e287-11eb-9221-1f8e0afb9296.png">

Is my approach make any sense?

How I can say if patients are similar or different from the controls? Any thoughts please?

**Comment by timoast** [comment]
If you want some tips for running a comparative analysis across datasets, please see the introductory integration vignette: https://satijalab.org/seurat/articles/integration_introduction.html
